src/components/postSubmit/AdditionalInformation.tsx

```diff
@@ -0,0 +1,73 @@
+type LinkType = {
+  title: string;
+  link: Record<string, string>;
+}
+
+type SubmitType = {
+  DescriptionAdditional: string;
+  TitleAdditionalInformation: string;
+  Feedback: LinkType;
+  Results: LinkType;
+}
+
+  function FeedbackLink({href, imageSrc, label}: {href:string, imageSrc:string, label:string}) {
+    if (href && imageSrc) {
+      return (
+        <a title={`${label} - ouvre une nouvelle fenêtre`} href={href}>
+          <img src={imageSrc} alt="" />
+        </a>
+      )
+    } else if (href) {
```

Comment:
else could be deleted here

Reply:
I would suggest an early return of null : 
```js 
if(!href) {
return null;
} 
if(href && imageSrc) { return (<a> with imageSrc)} 
return (<a> without imageSrc)
```
And agree, no need for else if for last return